meteor/client/lib/StyledTimecode.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Time, formatDurationAsTimecode } from '../../lib/lib'
+
+interface IProps {
+	time: Time
+}
+
+const FRAMES_INLINE_STYLE: React.CSSProperties = {
```

Comment:
Yeah, I specifically want this not to be dependent on the stylesheet, so that it's not overridden by any cascading styles. I realize it's not very pretty, but it ensures that I don't have to check if it actually works everywhere and modify the parent component's styling accordingly. I can just drop the StyledTimecode anywhere and it just works. I think it's an acceptable trade-off.